### DISCUSSION TITLE
Legacy skins in osu!mania it was weird (mixed keys) [8 keys mode]

### DISCUSSION THREAD

**Discussion by BKVad1m** [ppy/osu#19887] [Ideas]
this is mixed white and black keys example 2dx skin

![osu_2022-08-21_21-27-11](https://user-images.githubusercontent.com/76803825/185795949-a4f3fbe2-ca93-4e28-b240-58b1406a853c.jpg)

make color correction most like beatmania [Red, white, black, white, black, white, black, white]

see this original video and download
https://www.youtube.com/watch?v=9IrF7kYnDqY

**Comment by smoogipoo** [comment]
I believe this is because lazer doesn't support special styles at the moment: https://github.com/ppy/osu/issues/2203

Please provide the skin.

**Comment by BKVad1m** [comment]
https://user-images.githubusercontent.com/76803825/191918714-6183086f-d237-44fd-8188-b11c286eb943.mp4

I'm switching "Use Special (N+1) Style" in setup editor will not changed...
I think add feature in options "Use Always Special (N+1) Style" generating osu! to osu!mania

**Comment by BKVad1m** [comment]
this 8k mod can converted as special 7K+1K [MIDI Keyboard: F F# G G# A A# B] Special C [optional key binding]
